src/Symfony/Bundle/FrameworkBundle/EventListener/ConsoleProfilerListener.php

```diff
@@ -77,7 +77,7 @@ public function initialize(ConsoleCommandEvent $event): void
             return;
         }
 
-        $request->attributes->set('_stopwatch_token', substr(hash('xxh128', uniqid(mt_rand(), true)), 0, 6));
+        $request->attributes->set('_stopwatch_token', substr(hash('xxh128', random_bytes(14)), 0, 6));
```

Comment:
How did you choose to get `14` bytes?

`xxh3` algorithm is a little faster and returns a shorter string.
```suggestion
        $request->attributes->set('_stopwatch_token', substr(hash('xxh3', random_bytes(14)), 0, 6));
```

Reply:
14 was just a random guess. If we have more details, I can also choose another value.

For `hash()` I chose `xxh128` to be inline with what we use elsewhere since #52948

Reply:
oh, looks like there was a similar question :) https://github.com/symfony/symfony/pull/52948#issuecomment-1849604788

Reply:
Why do I need to hash a random value anyway? If what I want is to produce 6 random hex characters, how about:

```suggestion
        $request->attributes->set('_stopwatch_token', bin2hex(random_bytes(3)));
```

Reply:
Oh! Thanks for finding. It seems to be an obsession for me

At this point, since that's already random bytes and the generated string is hexadecimal characters, using `bin2hex` would be enough.

```suggestion
        $request->attributes->set('_stopwatch_token', bin2hex(random_bytes(3)));
```

Reply:
indeed, that's much better, PR updated